.github/workflows/ci.yml

```diff
@@ -3,11 +3,10 @@ on:
   workflow_dispatch: # Manual invocation.
   pull_request:
   push:
-    branches:
-      - main
-      - 'pr**'
 jobs:
   CI:
+    # only actually run if this is a pull_request from a fork OR push to an internal branch
+    if: github.event.pull_request.head.repo.fork || github.event_name != 'pull_request'
```

Comment:
what happens when this isn't a PR and you do `.pull_request.head`? are dots in GHA like `?.`s in JS, or will they throw some sort of "cannot get property of undefined" error? (if you swapped the order I think you wouldn't be able to get into that situation)

Reply:
I actually don't know, but I _think_ it behaves like optional chaining, but your suggestion is a good one, so I've updated